index.md

```diff
@@ -3,21 +3,23 @@ layout: default
 title: SapMachine
 ---
 
-**SapMachine** is a free, open-source, non-commercial, cross-platform, and fully-functional production-grade version of the Open Java Development Kit (OpenJDK).
-Built and supported by SAP, it provides a robust and reliable Java environment.
+**SapMachine** is the free, multiplatform, production-ready, and [Java SE certified](https://github.com/SAP/SapMachine/wiki/Certification-and-Java-Compatibility) [OpenJDK](https://openjdk.org/) (Open Java Development Kit) distribution by [SAP](https://sap.com). It serves as the default JDK for SAP's countless applications and services.
```

Comment:
SapMachine is one of many distributions of OpenJDK, so I would prefer something like:

```suggestion
**SapMachine** is a distribution of [OpenJDK](https://openjdk.org/) developed by SAP. It is designed to be free, cross-platform, production-ready, and [Java SE certified](https://github.com/SAP/SapMachine/wiki/Certification-and-Java-Compatibility). This distribution serves as the default Java Runtime for SAP's numerous applications and services.
```

or 

```suggestion
**SapMachine** is a free, cross-platform, production-ready distribution of [OpenJDK](https://openjdk.org/) (Open Java Development Kit) that is [Java SE certified](https://github.com/SAP/SapMachine/wiki/Certification-and-Java-Compatibility) by [SAP](https://sap.com). It serves as the default JDK for SAP's numerous applications and services.
```

Reply:
Oh, and while we are at it..., if you can agree would you mind changing the title, too? [https://github.com/SAP/SapMachine/blob/gh-pages/_config.yml](url)

It currently is:
`The OpenJDK release maintained and supported by SAP`

How about something a little more catchy?
`Fast. Reliable. Open`